src/homework/day8/Figure.java

```diff
@@ -0,0 +1,43 @@
+package homework.day8;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+public class Figure {
+    public static void main(String[] args) throws IOException {
+        List<String> figures = Arrays.asList("Овал", "Прямоугольник", "Круг", "Квадрат", "Эллипс");
+        BufferedWriter out = new BufferedWriter(new FileWriter("figures.txt"));
+        for (String figure : figures) {
+            out.write(figure + "-");
+        }
+        out.close();
+
+        int count = 0;
+        for (String figure : figures) {
+            if (!figure.contains("и")) {
+                count++;
+            }
+        }
+        System.out.println(count);
+
+        for (int i = 0; i < figures.size(); i++) {
+            System.out.print(figures.get(i) + " ");
+        }
+        try {
+            figures.add(3, "Треугольник");
+        } catch (RuntimeException error) {
```

Comment:
Could you please explain why did you implement here Exception?

Reply:
because I had the error and as per requirements we need to process it.